src/http/nameserver.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+import type { AxiosPromise } from 'axios';
+
+import { axiosGetDebug, axiosPostDebug } from './http-debug';
+
+const axios = require('axios');
+const login = require('../protocols/login');
+const kCont = require('../key-container/key-container');
+const proofs = require('../protocols/proofs');
+
+/**
+ * Class representing a the NameServer
+ * It contains all the relay API calls
+ * @param {String} url
+ */
+class NameServer {
+  url: string;
+  debug: boolean;
+  getFn: (string, any) => any;
+  postFn: (string, any) => any;
+
+  /**
+   * Initialization name server object
+   * @param {String} url - NameServer Url identifier
+   */
+  constructor(url: string, debug: boolean = false) {
+    this.url = url;
+    this.debug = debug;
+    if (this.debug) {
+      this.getFn = axiosGetDebug;
+      this.postFn = axiosPostDebug;
+    } else {
+      this.getFn = axios.get;
+      this.postFn = axios.post;
+    }
+  }
+
+  /**
+   * Construct proper object to bind an identity adress to a label
+   * Name resolver server creates the claim binding { Label - Identity address }
+   * @param  {Object} kc - Keycontainer
+   * @param  {String} idAddr - Identity address
+   * @param  {String} keyOperationalPub - Key used to sign the message sent
+   * @param  {String} name - Label to bind
+   * @return {Object} Http response
+   */
+  bindID(kc: kCont.KeyContainer, ksign: string, proofKSign: proofs.ProofClaim,
```

Comment:
I would name the function as: `bindId` instead of `bindID` just to follow name convention

Reply:
makes sense. done!